lib/board.ex

```diff
@@ -1,9 +1,13 @@
 defmodule Board do
   defstruct spaces: [0, 1, 2, 3, 4, 5, 6, 7, 8]
-  def create do
+  def create("3x3") do
     [0, 1, 2, 3, 4, 5, 6, 7, 8]
```

Comment:
While hard-coding this collection twice isn't the end of the world (sometimes I follow the rule of three, where I re-factor once something is repeated three times), is there a way to use a range to refactor this? If you pass in the board size instead of a string, you might not even need the pattern matching.